types/parse-prefer-header/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for node-persist 1.0
```

Comment:
Name of the package here should be `parse-prefer-header`

Reply:
Fixed in 3e11647639